app/components/entry/ExcerptInput/index.tsx

```diff
@@ -80,7 +90,69 @@ function ExcerptInput<N extends string>(props: Props<N>) {
                         {value}
                     </div>
                 )}
-            </div>
+            </Container>
+        );
+    }
+    // Manually added images (e.g. using screenshot)
+    if (entryType === 'ATTACHMENT') {
+        const filePreview = leadAttachment
+            ? leadAttachment.filePreview
+            : entryAttachment?.filePreview;
+
+        const fileType = leadAttachment
+            ? leadAttachment.type
+            : entryAttachment?.entryFileType;
+
+        const file = leadAttachment
+            ? leadAttachment.file
+            : entryAttachment?.file;
+
+        const attachmentSrc = imageRaw ?? filePreview?.url;
+        return (
+            <Container
+                className={_cs(className, styles.excerptInput, styles.imageExcerptContainer)}
+                headerActions={fileType === 'XLSX' ? (
+                    <QuickActionLink
+                        title="Open external"
+                        to={file?.url || ''}
+                    >
+                        <BsDownload />
+                    </QuickActionLink>
+                ) : undefined}
```

Comment:
Don't use empty string as default value, also this expression can be simplified with `&&`
```suggestion
                headerActions={fileType === 'XLSX' && isDefined(file?.url) &&  (
                    <QuickActionLink
                        title="Open external"
                        to={file.url}
                    >
                        <BsDownload />
                    </QuickActionLink>
                )}
```